### ISSUE TITLE
Missing/unresized images thread

### ISSUE THREAD

**Issue by drojf** [07th-mod/umineko-answer#15]
I think all the panningg images need to be fixed

Both of these are from the same scene:
![honeyview_bug2](https://user-images.githubusercontent.com/1249449/34588329-86541bf6-f200-11e7-8248-6257b7702855.jpg)

- Stake effect during `I greet you in the name of the seven sisters of purgatory` - **all stake effects should be fixed now**

EDIT butterflies should be fixed by new images, panning shot should be fixed

**Comment by ctom42** [comment]
Blue truth effects just prior to this line were not sized properly. 
![bug6-loc](https://user-images.githubusercontent.com/2365513/34588858-3138e454-f17d-11e7-9382-bded1979d436.png)

**DROJF - I think this should be fixed in the next graphics update - some assets weren't included which should have been there.**

**Comment by ctom42** [comment]
Two voice lines did not play in this image
![bug7](https://user-images.githubusercontent.com/2365513/34589501-0ca265be-f17f-11e7-9c64-b5d77031200f.png)

"She never saw it." worked fine. "The room ERika was assigned to." and "Where was it located?" did not play. 

I tested twice to make sure it consistently failed to play. I then checked it in Japanese and it appears Virgilia's text is only in two chunks rather than 3 in Japanese. This means there is likely only one voice line missing rather than two.

Interestingly enough, this difference in the amount of clicks needed to advance the text resulted in Battler speaking the first half of Virgilia's line and Virgilia speaking the second. 
  
  **DROJF - Fixed the unvoiced line, but couldn't reproduce your 'battler speaking virgilia's line' bug**

**Comment by ctom42** [comment]
Another vertical cathedral pan just before this line. Might very well be the same one as the first one I reported, but I wanted to point it out just in case it's not

![bug8-loc](https://user-images.githubusercontent.com/2365513/34589782-9a43e518-f180-11e7-809c-8780c3a5f1cb.png)

 **DROJF - all the vertical pans should be fixed, let me know if you find any which are not fixed**
  
some other pans I found:

(cathedral pan up needs to be fixed)
[honeyview_bug1](https://user-images.githubusercontent.com/1249449/34588330-86909176-f200-11e7-8d94-75163d15b68d.jpg)

(other pan)
  [honeyview_bug3](https://user-images.githubusercontent.com/1249449/34588494-62e0134a-f201-11e7-90d1-ed9bfb6bb15c.jpg)

**Comment by ctom42** [comment]
Unreplaced sprite. It appeared a few lines before this, but I was skipping so I didn't catch it immediately

![bug9](https://user-images.githubusercontent.com/2365513/34589838-f9845d78-f180-11e7-9989-15cb05ecc1b5.png)

**DROJF - Fixed all instances of `mldg` function where character appears in transparent form - should fix this issue. There were ~50 instances of this in the script.**
  
Another one. I suspect it's not the sprites themselves that are a problem, but rather the code that calls for them when they are transparent.

[bug10](https://user-images.githubusercontent.com/2365513/34589941-c18e9a22-f181-11e7-9a33-123bdcceb04d.png)

  **DROJF - Fixed all instances of `mldg` function where character appears in transparent form - should fix this issue**

**Comment by ctom42** [comment]
Another one. I suspect it's not the sprites themselves that are a problem, but rather the code that calls for them when they are transparent.

![bug10](https://user-images.githubusercontent.com/2365513/34589941-c18e9a22-f181-11e7-9a33-123bdcceb04d.png)

**Comment by ctom42** [comment]
Red sword slashes just before this line were not the right size
![bug11-loc](https://user-images.githubusercontent.com/2365513/34589986-fb0a7d48-f181-11e7-971c-8932f80c829e.png)

**Comment by ctom42** [comment]
Gertrude's red shield
![bug12](https://user-images.githubusercontent.com/2365513/34590015-24a48c70-f182-11e7-9583-72ae7acfa9eb.png)

**Comment by ctom42** [comment]
Another, different shield effect
![bug13](https://user-images.githubusercontent.com/2365513/34590180-07b965f8-f183-11e7-8600-cdcbe93d02b3.png)

This is the line that follows it
![bug13-loc](https://user-images.githubusercontent.com/2365513/34590188-128847d8-f183-11e7-85a1-3fe5fc082df6.png)

Interestingly, a few lines later, an extremely similar effect appears but is not bugged. I'm wondering if maybe these are supposed to both be the same effect, or if there are just more different versions of the effect than I was aware of.
![not-bugpng](https://user-images.githubusercontent.com/2365513/34590194-1e7d757c-f183-11e7-89f3-eb53f0a6e5bd.png)

**Comment by ctom42** [comment]
More missing red sword effects right before this line, might be the same ones as before, not sure. 

![bug14](https://user-images.githubusercontent.com/2365513/34590340-10f5d218-f184-11e7-9733-bbc63da8cc15.png)

Also worth noting that having only the end quote on it's own line is very awkward.

**Comment by ctom42** [comment]
Ok so, after the second "FINISHED", aka the end of the first line, the rest of the text does not appear until Dlanor is finished saying all of it, and then it just appears all at once. 
![bug15](https://user-images.githubusercontent.com/2365513/34590478-0a9b1922-f185-11e7-9de4-5215ef6e5263.png)
Also, perhaps more importantly, the lines "DIE THE DEATH! SENTENCED TO DEATH! GREAT EQUALIZER IS THE DEATH!" are mostly off the screen. They too don't appear until Dlanor is finished with all of them, and then only a small portion of the top of the first one is visible, and the scene quickly transitions away so even that is only there for a fraction of a second.

**DROJF - Fixed the text not appearing until voice is finished (for ALL automatically played voice lines, not just this one) . Also for dlanor's speeches particularly, fixed the voices being cut off.**

**Comment by ctom42** [comment]
Installed all the updates and checked the last two chapters of episode 5 again. Here's what I found

First, the clock needs resizing. I noticed this last time but forgot to post an image since i was told it was a known bug.
![clock](https://user-images.githubusercontent.com/2365513/34655264-b79776a6-f3d4-11e7-8db8-aa03c82eb750.png)

**DROJF - Fixed the clock by positioning it correctly for 1080p. The image hasn't been modified/scaled otherwise, just re-positioned (modified the function which draws the clock, so should be a global fix)**

I found a missing BG that I missed in my previous pass
![missing bg](https://user-images.githubusercontent.com/2365513/34655272-e778bc9a-f3d4-11e7-840f-6c413cf72927.png)

**DROJF - Fixed the 'missing bg' and a couple more instances of something similar in the script. haven't fixed the other stuff yet**

----

- Next, the butterfly effect as seen in the first image in this thread is still broken. 

- Also the first Gertrude shield effect is broken, but the later one appears fixed.
